content/roadmaps/114-defi/content/111-yield-optimizers/108-beefy.md

```diff
@@ -1 +1,7 @@
 # Beefy
+
+Beefy is a decentralized, multichain yield optimizer that allows its users to earn compound interest on their crypto holdings. It supports more than a dozen of different EVM blockchains.
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://docs.beefy.finance/'>Beefy Docs</BadgeLink>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://beincrypto.com/learn/beefy-finance/'>What Is Beefy Finance?</BadgeLink>
```

Comment:
Example alternative: https://messari.io/report/beefy-finance-multichain-yield-optimizer (probably not the best)

Goal should be to include as much quality technical articles as possible 😉

Reply:
Is this proper link ? Getting "Internal Server Error"

Reply:
Yes, work well know. Maybe some temp issue